packages/react/dialog/src/Dialog.stories.tsx

```diff
@@ -521,7 +512,7 @@ const animatedOverlayClass = css(overlayClass, {
 
 const animatedContentClass = css(contentDefaultClass, {
   '&[data-state="open"]': {
-    animation: `${fadeIn} ${scaleIn} 300ms ease-out`,
+    animation: `${fadeIn} 150ms ease-out, ${scaleIn} 200ms ease-out`,
```

Comment:
This is another change I made, not sure if it ever worked, but I noticed it definitely didn't with this new version (ie. you need 2 separate animations, comma separated).